components/pagination/style/index.less

```diff
@@ -115,9 +115,9 @@
     color: @text-color;
     border-radius: @border-radius-base;
     list-style: none;
-    min-width: 28px;
-    height: 28px;
-    line-height: 28px;
+    min-width: @input-height-base;
```

Comment:
@benjycui The second commit reverts the pagination component [https://github.com/ant-design/ant-design/pull/4175/commits/d90e14eacf376d98ad36653d2f09007da019ea1d](https://github.com/ant-design/ant-design/pull/4175/commits/d90e14eacf376d98ad36653d2f09007da019ea1d)

Is this not correct?